src/index.html

```diff
@@ -17,6 +17,92 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
+        <ul class="navi-menu">
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link is-active"
```

Comment:
The class name 'navi-link' could be simplified to 'nav__link' to follow the BEM naming convention. Additionally, the 'is-active' modifier should be written as 'nav__link--active' to follow the BEM naming convention.